pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    slow: Tests that are slow and may require more resources
+    mpl_image_compare
```

Comment:
These changes should be added to the `pyproject.toml` file, in this [section](https://github.com/astropy/astropy/blob/5b79a7c2c49a52868703b6e0cb0016aa5d6cd312/pyproject.toml#L247-L249), not added to its own separate configuration file.